probe-rs/src/core/mod.rs

```diff
@@ -382,6 +387,15 @@ impl SpecificCoreState {
                 crate::architecture::arm::armv6m::Armv6m::new(memory, s, debug_sequence)?,
                 state,
             ),
+            SpecificCoreState::Armv7a(s) => Core::new(
+                crate::architecture::arm::armv7a::Armv7a::new(
+                    memory,
+                    s,
+                    base_address.unwrap(),
```

Comment:
Replaced unwrap with an error.

This state cannot be on Target because it is per core.  This is how SMP on v7-A works - each CPU core gets its own bank of debug registers and you tell the debugger each location per core so it knows how to control them.